src/composables/widgets/useComboWidget.ts

```diff
@@ -0,0 +1,414 @@
+import { ref } from 'vue'
```

Comment:
For primvevue widgets: load 100 up front, batches of 20
For litegraph widgets: can we do pagination effectively or click to load more?

Reply:
See if we merged assetStore because we can piggyback off that

Reply:
cache asset map in store

Reply:
see if we can inherit off the combo widget